tests/cypress/e2e/header.cy.js

```diff
@@ -88,7 +88,7 @@ describe('Header - Item and Manifest changing', () => {
     cy
       .get(selectors.nextButton)
       .should('not.be.disabled')
-      .contains('Next Sheet');
+      .contains('Next Item');
```

Comment:
This should actually stay as before. Ahiqar should now also change it's configration towards the new keys since there a manifest is called "Manuscript" and item is called "Sheet". In fact, Manuscript and Sheet were always Ahiqar-specific namings and had nothing to do with other projects.

Reply:
does that mean that we should create another 4 new keys only for Ahiqar, like 'Next Sheet', 'Previous Sheet', 'Next Manuscript' and 'Previous Manuscript' ?